### ISSUE TITLE
Web component and React: Locale modal -  Border of the Search field (Color Contrast)

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4801]
## Detailed description

Actual Result:
All sides of borders on the search field is not meeting the success contrast ratio
Foreground:#F4F4F4
Background:#FFFFFF

The contrast ratio is: 1.1:1
Text failed at Level AA

Expected Result:
Provide below contrast ratio.
Foreground:#8A8A8A
Background:#FFFFFF

The contrast ratio is:3.5:1
Text passed at Level AA

**Comment by wonilsuhibm** [comment]
### Border
Now sure if this is an issue. The borders are optional as we communicate via the underline per Carbon specs.

### Text

Is the text also an issue? I see the text having the right color to pass a11y.
<img width="1243" alt="Screen Shot 2021-01-19 at 1 15 07 PM" src="https://user-images.githubusercontent.com/30183447/105076015-892d4780-5a58-11eb-9868-babc767c8325.png">

<img width="502" alt="Screen Shot 2021-01-19 at 1 17 23 PM" src="https://user-images.githubusercontent.com/30183447/105076132-af52e780-5a58-11eb-8997-abeba5e94518.png">

**Comment by RobertaJHahn** [comment]
Praveen will confirm with Ram that the existing underline is OK and will post a response to this issue.

**Comment by Praveen-sr** [comment]
@andysherman2121 Since the underline is bold and dev team is making the placeholder also visible for the user therefore we can ignore the all side order defect.

CC @RobertaJHahn
